packages/xstate-wallet/src/integration-tests/running.test.ts

```diff
@@ -30,7 +30,7 @@ test('accepts states when running', async () => {
   ]);
 
   hookUpMessaging(playerA, playerB);
-  const channelId = '0x1823994d6d3b53b82f499c1aca2095b94108ba3ff59f55c6e765da1e24874ab2';
+  const channelId = '0x440b56b6c5b0adca1ee99e3926d1b123fd867566cfab5150479f9d5e9317fa1e';
```

Comment:
How are these things calculated? Maybe add a note

Reply:
I actually don't really know 🤷. The test started failing when we started properly reading in `.env.test` env vars, which changed the chain network id.